### ISSUE TITLE
Implement Rotate Letter Screen

### ISSUE THREAD

**Issue by thellmund** [serlo-org/serlo-abc#20]

**Comment by SimonKoehl** [comment]
Moved to Version 1.x according to planning last friday

**Comment by ronnitatserlo** [comment]
![17](https://cloud.githubusercontent.com/assets/20284270/24862427/ce66ed0c-1dfc-11e7-9d2c-a7a1817a7c0b.jpg)

**Comment by zepatrik** [comment]
So I don't have to do it know, it will be added in some time right?

**Comment by inyono** [comment]
@zepatrik "Version 1.x" means that it is not planned for the first release, but soon after. So if you already did some work, feel free to continue. Otherwise, you can pick some issue in the "Version 1.0" column :)

**Comment by zepatrik** [comment]
All right, I started but ran into issues with react-native-svg

**Comment by inyono** [comment]
Yeah, since we use [CRNA](https://github.com/react-community/create-react-native-app), we can't use packages that require linking (like `react-native-svg`) directly. But for this we get a much better development experience with expo etc. We can eject later and somehow use such native modules but if possible I want to manage without that.

You can push your current branch if you want me to look over it. I'm pretty sure that we don't need react-native-svg for this exercise.
